test/e2e/framework/device_plugin_util.go

```diff
@@ -0,0 +1,76 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package framework
+
+import (
+	appsv1 "k8s.io/api/apps/v1"
+	"k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/serializer"
+	"k8s.io/kubernetes/test/e2e/framework/testfiles"
+
+	"github.com/onsi/ginkgo"
+)
+
+const (
+	// sampleResourceName is the name of the example resource which is used in the e2e test
+	sampleResourceName = "example.com/resource"
+	// sampleDevicePluginDSYAML is the path of the daemonset template of the sample device plugin. // TODO: Parametrize it by making it a feature in TestFramework.
+	sampleDevicePluginDSYAML = "test/e2e/testing-manifests/sample-device-plugin.yaml"
```

Comment:
nit: How about adding `sampleDevicePluginName` here like `sampleDevicePluginName = "sample-device-plugin"` and using it at line 58?